build glog with cmake error

when building glog with cmake following the step in readme after conducting the 3rd step 3. Afterwards, generated files can be used to compile the project by command `cmake --build build`, error occurs as following:
```console
[  3%] Building CXX object CMakeFiles/cleanup_with_absolute_prefix_unittest.dir/src/cleanup_with_absolute_prefix_unittest.cc.o
make[2]: *** No rule to make target 'CMakeFiles/glog_internal.dir/src/demangle.cc.o', needed by 'cleanup_with_absolute_prefix_unittest'.  Stop.
make[1]: *** [CMakeFiles/Makefile2:128: CMakeFiles/cleanup_with_absolute_prefix_unittest.dir/all] Error 2
make: *** [Makefile:163: all] Error 2
```
not really sure why this occurs and how to solve it?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.16)
+cmake_minimum_required (VERSION 3.21)
 project (glog
   VERSION 0.7.0
   DESCRIPTION "C++ implementation of the Google logging module"

